packages/pwa-kit-create-app/assets/bootstrap/js/package.json.hbs

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "{{answers.project.name}}",
+  "version": "0.0.1",
+  "license": "See license in LICENSE",
+  "engines": {
+    "node": "^16.0.0 || ^18.0.0",
+    "npm": "^7.0.0 || ^8.0.0 || ^9.0.0"
+  },
+  "ccExtensibility": {
+    "extends": "{{preset.templateSource.id}}",
+    "overridesDir": "overrides"
+  },
```

Comment:
A project generated with the answer "No" to the question: Do you wish to use extensibility? still includes in the `package.json` file the key:

```
"ccExtensibility": {
    "extendable": [
      "retail-react-app"
    ]
  },
  ```
  
  Is that expected?

Reply:
Addressed!

Reply:
I can confirm this is fixed

Reply:
@adamraya this is required because it hooks into the magic in `pwa-kit-dev` for aliasing the base path `retail-react-app/` => `<projectRoot>`

Apologies for the confusion

Reply:
@bfeister I don't see this as a blocker but from the end user perspective, it's confusing that you generate a project saying 'No' to extensibility and then you find a key in your package.json file with the value `extendable`.

How will the end user tell the difference between a real extendable project from one with the answer 'No' with the `extendable` key?

Perhaps we can adapt the Webpack config ternary conditions to address the scenario that we want to support.

Reply:
@adamraya totally agree. I think we need to start "cutting" things though, so I'd like to address this (personally, I'd rather it be a find/replace in the generator's "eject" action to replace `retail-react-app/app/*` with `app/*` 

This was born out of the fact that the idea was that Template Extensibility would become the default